### PR TITLE
adding heart disease prediction

### LINKED ISSUE #861: Heart disease prediction
**This template is strictly for requesting New Scripts, if you want a new feature (automating this Repo,**
**a change in structure) in the Repo,** 
**or you want to report a Bug. There are other templates which you should follow**

# Description

Please include a summary of the issue. 
This is machine learning model trained using ucl heart disease dataset , the script predicts whether a person has heart disease or not by inputing health condition of the patient
## Language

- [x] Python

## Checklist:

- **Details on the Script**

The script predicts whether a person is affected from heart disease or not

heart disease predictor

| name            | about                                                  | labels  | assignees                                                      |
| --------------- | ------------------------------------------------------ | ------- | -------------------------------------------------------------- |
| Heart disease predictor | predicts whether a person has heart disease or not | gssoc'21 | @aaadddiii |

**Is your feature request related to a problem? Please describe.**: A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

**Describe the solution you'd like**: A clear and concise description of what you want to happen.

**Additional context**: Add any other context or screenshots about the feature request here.

### PR DESCRIPTION
**Pull request by aaadddiii**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

Fixes #861 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

File: Python/Heart Disease Prediction/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+```
+Install the requirements 
+```bash
+pip install -r requirements.txt
+```
+## To run
+```bash
+python script.py
+```
+#### By [Adithya Kurien Peter]("https://github.com/aaadddiii")
```
Comment (vybhav72954) [inline Python/Heart Disease Prediction/README.md:23]: Is this correct?

Comment (aaadddiii) [reply Python/Heart Disease Prediction/README.md:23]: It works

Comment (vybhav72954) [reply Python/Heart Disease Prediction/README.md:23]: The Author line, is that correct?

Comment (aaadddiii) [reply Python/Heart Disease Prediction/README.md]: It was a mistake, sorry . I've made necessary changes

File: Python/Heart Disease Prediction/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+```bash
+pip install -r requirements.txt
+```
+## To run
+```bash
+python script.py
+```
+## Author
+ 
+[Adithya Kurien Peter](https://github.com/aaadddiii)
```
Comment (vybhav72954) [inline Python/Heart Disease Prediction/README.md:25]: Dear @aaadddiii your script shares similarity with [this](https://github.com/mrdbourke/zero-to-mastery-ml/tree/master/section-3-structured-data-projects).

I am all in for inspired code. I have done the same many times, but am against using someone else's code and claiming it as it's your own.

You can do either of these 3 things

1. in the README mention the exact name of the Author
2. Make some changes in the existing code to make it better
3. Add some personal touches to the script

Comment (aaadddiii) [reply Python/Heart Disease Prediction/README.md:25]: Sure

Comment (aaadddiii) [reply Python/Heart Disease Prediction/README.md:25]: The similarity of the code is that i learned machine learning from his course this is a project in his course . i'l mention his name in the README .

Comment (vybhav72954) [reply Python/Heart Disease Prediction/README.md]: That seems good enough to me.

Kindly make the changes in the README, make sure that you properly mention the source as well as your contribution.

**Comment by aaadddiii** [comment]
necessary changes are made

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Sorry but we don't host ML Models here unless they provide useful for us. Thanks for your contributions!

**Pull request closed without merging**